model.js

```diff
@@ -61,3 +61,19 @@ exports.selectComments = (articleID) => {
       }
     });
 };
+
+exports.writeContent = (author, body, article_id) => {
+  return db
+    .query(
+      "INSERT INTO comments (author, body,  article_id) VALUES ($1, $2, $3) RETURNING *",
+      [author, body, article_id]
+    )
+    .then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
+      if (err.code === "23503") {
+        return Promise.reject("invalid input");
+      }
```

Comment:
We should not have a .catch(...) in the model file. This should be send directly to your .catch(...) in your controller which sends the error to your middleware error handlers which should deal with this